scripts/deployment/create_incentives.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+deployment_script_dir=$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)
```

Comment:
explain BASH_SOURCE, is it just the source of this script ?

Reply:
That's the path of the script being executed